app/assets/stylesheets/equipment_models/_show.css.scss

```diff
@@ -132,14 +132,15 @@ section {
 
 a.not-qualified-icon-em {
 	position: absolute;
-	top: 3px;
+	top: 0;
 	right: 120px;
-	font-size: $baseFontSize * 3;
-	color: $btnWarningBackgroundHighlight;
+	font-size: ceil($font-size-base * 2.5);
+  line-height: 1em;
+	color: $btn-warning-bg;
 	@include warning-icon-drop-shadow;
 
 	&:hover, &:active {
 		text-decoration: none;
-		color: $btnWarningBackground;
+		color: $btn-warning-bg;
```

Comment:
Isn't this redundant now that we're setting the nominal color to `$btn-warninig-bg` (line 139)?